algebra/src/rand.rs

```diff
@@ -16,3 +16,14 @@ where
         rng.sample(Standard)
     }
 }
+
+pub fn test_rng() -> rand::rngs::StdRng {
```

Comment:
~We can put this under `#![cfg(test)]` so that it's not exported.~

I see that it's used in `ff-fft` and other crates. Maybe we can add a line of documentation indicating that this should be used only for testing purposes, and not for any real-world usage?